ratings/views.py

```diff
@@ -1,3 +1,67 @@
-from django.shortcuts import render
+import json
+from json.decoder import JSONDecodeError
 
-# Create your views here.
+from django.http  import JsonResponse
+from django.views import View
+
+from ratings.models import StarRating
+from users.models   import User
+from cafes.models   import Cafe
+from utils          import LoginConfirm
+
+class StarRatingView(View):
+    @LoginConfirm
+    def post(self, request, cafe_id):
+        try:
+            data = json.loads(request.body)
+
+            user = User.objects.get(id=request.user.id)
```

Comment:
이미 데코레이터를 통해 검증된 사용자의 user 를 전달받고 있기 때문에, 다시 유저정보를 요청하지 않아도 괜찮습니다.
```suggestion
            user = request.user
```